app/modules/topics/components/TopicsList/TopicCard/index.js

```diff
@@ -119,7 +119,11 @@ class PureTopicCard extends React.Component<Props, ComponentState> {
       <React.Fragment>
         <Card data-test-id="topic-card">
           <Card.Content>
-            <Card.Header>{topic.title}</Card.Header>
+            <Card.Header>
+              <Link to={makeRoute(TOPIC_COURSE_ROUTE, { topicId: topic.id })}>
+                {topic.title}
+              </Link>
+            </Card.Header>
```

Comment:
Could you add this in the topicCardButtons instead of as a link on the topic card header? Because I actually had to check the source code in order to find it so I'm going to assume a lot of other users would be equally as confused 😅

Reply:
You mean adding _View_ as a button? I specifically added this on the title, because it is more intuitive and I kept clicking on the title to go to the topic.

Reply:
Hmm, I guess we just have diffent opinions about what is intuitive then? I was very confused by it because in a card with buttons I assume all of the available actions can be found in the buttons; a link on the header (or on the whole card) is generally only done when there is only one action, or at least a clear 'main' action.

Reply:
Solution: put it both in the buttons and as a link on the header? :P